src/css/styles.scss

```diff
@@ -45,6 +47,7 @@
     image-rendering: -moz-crisp-edges; // FF
 
     z-index: var(--zIndex-canvas);
+    position: absolute;
```

Comment:
```suggestion
    // remove canvas from document flow to avoid resizeObserver feedback loop
    // (see https://github.com/excalidraw/excalidraw/pull/3379)
    position: absolute;
```